app/src/main/contrib/components/Responsibilities/Overview/ConfidentialityAgreement.tsx

```diff
@@ -31,22 +31,29 @@ export class ConfidentialityAgreementComponent extends React.Component<Confident
     }
 
     render() {
-        return <div className={"row"}>
-            <div className={"col-12 col-md-6 offset-md-3"}>
-                <div className={"border p-3 border-dark mb-5"}>
-                    I have read and understood the terms of the <a href={fullConfidentialityAgreement}>
-                    RfP applicants' confidentiality agreement</a>. In doing so, I understand and agree not to disclose
-                    or share any information on vaccine coverage data which I access from Montagu,
-                    beyond my immediate RfP modelling group.
-                    <input type={"checkbox"} className={"mt-2 mb-2 d-block"}
-                           style={{height: "20px", width: "20px"}} checked={this.state.checked}
-                           onChange={this.onChange}/>
-                    {this.state.checked &&
-                    <button className="btn-success" onClick={this.props.signAgreement}>Submit</button>
-                    }
+
+        if (this.props.signed == null) {
+            return <LoadingElement />
+        }
+        else {
+            return <div className={"row"}>
```

Comment:
Maybe add a comment here explaining that `signed` must be `false` this point, because if it was `true` the branch would have hidden this component? And then I'm happy for this to merge (assuming TeamCity passes)